### PR TITLE
Update CacheInterface.php

### PR DESCRIPTION
**Pull request by polishExperiment**

### CODE REVIEW

**Review by dragoonis — APPROVED** [review]
I approve this typo fix. Thanks for pointing it out.

**Comment by dragoonis** [comment]
@Jean85 I have no merge rights on this repo, anymore (i think?). Can you please merge this, once it's safe to do so.

**Review by Jean85 — APPROVED** [review]
This is just a typo fix, so it's fine to be merged (and even released as a patch) without further consideration :+1: 

Thanks for spotting it.

**Pull request merged**
